app/Services/GithubService.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Based on the provided diff, I found a few issues in `app/Services/GithubService.php`:

1. Missing `use` statement for `\GuzzleHttp\ClientFactory` is present but not required.
2. The class does not follow PSR-12 coding standards for naming conventions and spacing.

Overall, the code looks well-structured, and no critical issues were found. LGTM.